dotcom-rendering/src/components/Accessibility.importable.tsx

```diff
@@ -67,19 +106,23 @@ export const Accessibility = () => {
 		const timeout = setTimeout(() => {
 			// we must reload the page for the preference to take effect,
 			// as this relies on a server-side test & cookie combination
-			if (participate !== darkModeAvailable) window.location.reload();
+			if (shouldParticipate !== darkModeAvailable) {
+				window.location.reload();
+			}
 		}, 1200);
 
 		return () => clearTimeout(timeout);
-	}, [participate, darkModeAvailable]);
+	}, [shouldParticipate, darkModeAvailable]);
 
-	const toggleFlash = (): void => {
-		setShouldFlash((prev) => (isUndefined(prev) ? false : !prev));
+	const togglePreference = (
+		preferenceCallback: (value: SetStateAction<boolean>) => void,
+	): void => {
+		preferenceCallback((prev) => (isUndefined(prev) ? false : !prev));
 	};
 
 	return (
 		<FrontSection title="Preferences" editionId="UK" discussionApiUrl="">
-			<form>
```

Comment:
Why is this removed?

Reply:
Initial thought was that we're not actually submitting here, but as we still dispatch on check I've added this back in.